setup.py

```diff
@@ -7,6 +7,11 @@
 
 from setuptools import setup, find_packages
 
+# Make the current nilearn sources importable.
```

Comment:
I would rather that you use the mechanism used below in "get_version", that doesn't rely on importing nilearn. Indeed, there are multiple reasons why importing nilearn could fail with a non helpful error, while simply running the corresponding file (as in get_version), is quite robust.